KazuateQuiz/src/kazuate/KazuateQuiz.java

```diff
@@ -1,18 +1,27 @@
 package kazuate;
 
-import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Random;
 
 public class KazuateQuiz {
 	public static void main(String[] args) {
+
 		//ゲームの説明
 		System.out.println("0～9でランダムに生成される3つの数のうち、どれか1つの数字を当ててください。");
 		System.out.println("5回まで挑戦できます。");
 		System.out.println("では、スタート!" + "\n");
-		//生成されたランダム値
-		int[] numbers = comanswer();
-		//answer が true になったら数あてクイズが終了する。または、ループが5回まわったらゲームを終了する。
-		boolean answer = false;
+
+		//ランダム値 0～9 を含む、int型の配列生成
+		int[] numbers = new Random().ints(3, 0, 9).toArray();
+
+		//int型配列をリストへ変換
+		List<Integer> generateNumbers = new ArrayList<>(numbers.length);
```

Comment:
`generateNumbers`という命名だと動詞 + 名詞になって数を生成するみたいな意味になるのでメソッドっぽいです！
今回でいくと、数のListなので単純に`numbersList` とかの方が素直な感じがするのですがいかがでしょうか？🤔

Reply:
```suggestion
        List<Integer> numberList = new Random().ints(3, 0, 9).collect(ArrayList::new, ArrayList::add, ArrayList::addAll);
```

不細工ではありますが、L.16とL.19をワンライナーで書くこともできるので参考までに！

Reply:
メソッド名、変数名を付けるのが下手なようです…。
`generateNumbers`を翻訳にかけたら「数字を生成する」になったので、確かにメソッドっぽいと感じました。
ご提案いただいた通り`numbersList`が適切だと思いましたので、`numbersList`へ修正しました。
ご確認をお願いいたします。
54acd043248ed62661049ea748ad13ad97e356b1